admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -0,0 +1,382 @@
+<!--**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *-->
+<template>
+  <div id="product-images-container">
+    <div
+      id="product-images-dropzone"
+      :class="['dropzone', 'dropzone-container', { full: files.length <= 0 }]"
+    >
+      <div
+        :class="[
+          'dz-preview',
+          'openfilemanager',
+          { 'd-none': loading || files.length <= 0 },
+        ]"
+      >
+        <div>
+          <span><i class="material-icons">add_a_photo</i></span>
+        </div>
+      </div>
+      <div
+        :class="[
+          'dz-default',
+          'dz-message',
+          'openfilemanager',
+          'dz-clickable',
+          { 'd-none': loading || files.length > 0 },
+        ]"
+      >
+        <i class="material-icons">add_a_photo</i><br>
+        {{ $t('Drop images here') }}<br>
+        <a>{{ $t('or select files') }}</a><br>
+        <small>
+          {{ $t('files recommandations') }}<br>
+          {{ $t('files recommandations2') }}
```

Comment:
```suggestion
          {{ $t('files recommendations') }}<br>
          {{ $t('files recommendations2') }}
```

Reply:
It's from the old product page, asked @Julievrz if we can change every keys to follow a pattern